eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -338,4 +338,30 @@
     <Field name="~(MAX_DURATION|NULL_TXN)"/>
     <Bug pattern="MS_SHOULD_BE_FINAL"/>
   </Match>
+
+  <!--  The return value is ignored for purpose-->
+  <Match>
+    <Class name="class com.microsoft.azure.servicebus.primitives.AsyncUtil"/>
+    <Method name="~(completeFuture|completeFutureExceptionally)"/>
+    <Bug pattern="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE"/>
+  </Match>
+
+  <!--  The casting is correct, it is beyond the SpotBugs's ability to determine the correctness -->
+  <Match>
+    <Class name="com.microsoft.azure.servicebus.primitives.RequestResponseUtils"/>
+    <Method name="encodeRuleDescriptionToMap"/>
+    <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
+  </Match>
+
+  <!--  Bytes array are designed to return null if it is null-->
+  <Match>
+    <Class name="com.microsoft.azure.servicebus.Utils"/>
```

Comment:
[SpotBugs-P3]

Suppress it because the function returns null means somethings

Should com.microsoft.azure.servicebus.Utils.getDataFromMessageBody(MessageBody) return a zero length array rather than null?